composeApp/src/commonMain/kotlin/sp/bvantur/tasky/core/ui/components/TaskyInitialsCircle.kt

```diff
@@ -0,0 +1,37 @@
+package sp.bvantur.tasky.core.ui.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun TaskyInitialsCircle(value: String, modifier: Modifier = Modifier) {
+    // TODO handle multiple names, emojis and wierd characters
+    val initials = value.split(" ")
+        .filter { it.isNotBlank() }
+        .take(2)
+        .joinToString("") { it.first().uppercase() }
```

Comment:
Would this logic not be best located in the domain layer? Placing it here makes this composable function responsible for a business-related task, namely creating a pair of initials based on some rules that can be altered. And when those rules change, you’ll need to update your UI code. 

Placing this logic in your UI makes testing harder, as you can’t just write a simple (and lightweight) unit test.